firmware/config/config_helper.h

```diff
@@ -1,6 +1,6 @@
 // Include config.h if it exists
```

Comment:
nit: you did not update the name in this comment

Reply:
fixed